Makefile

```diff
@@ -83,13 +96,16 @@ deps:
 docker-images: build-unix
 	docker build -t $(API) -f build/package/$(API).Dockerfile .
 	docker build -t $(PROXY) -f build/package/$(PROXY).Dockerfile .
+	docker build -t $(PROXY_CARTHAGENET) -f build/package/$(PROXY_CARTHAGENET).Dockerfile .
 docker-tag:
 	docker tag $(API) ${REGISTRY}:$(API)-dev
 	docker tag $(PROXY) ${REGISTRY}:$(PROXY)-dev
+	docker tag $(PROXY_CARTHAGENET) ${REGISTRY}:$(PROXY_CARTHAGENET)-dev
 docker-push:
 	echo ${PASSWORD} | docker login -u ${USERNAME} --password-stdin
 	docker push ${REGISTRY}:$(API)-dev
 	docker push ${REGISTRY}:$(PROXY)-dev
+	docker push ${REGISTRY}:$(PROXY_CARTHAGENET)-dev
```

Comment:
Technically, this is the same image here. It will just create duplication on the registry.

Reply:
agreed